wikidatarefisland/external_identifiers/generate_whitelisted_ext_ids.py

```diff
@@ -67,7 +67,7 @@ def run(self):
 
         pids = []
         for i in external_identifiers:
-            if i in final_results or i in self.config.get('blacklisted_properties'):
+            if i in final_results or i in self.config.get('blacklisted_external_identifiers'):
```

Comment:
I'm not sure I understand, why was the config being read here was "blacklisted_properties" to begin with? Shouldn't those be taken into account at a different level, namely in pipe1 when deciding which statements needs references?

Reply:
Yeah but they all share one config module. It was a mistake by yours truly to use the wrong config here.

Reply:
Sure thing :) Thanks for the answer.